package.json

```diff
@@ -48,9 +49,11 @@
     "typescript": "^2.1.5"
   },
   "dependencies": {
+    "@types/react-router": "^3.0.6",
     "asap": "https://github.com/alloy/asap.git",
     "react": "^15.4.2",
     "react-relay": "https://github.com/alloy/relay/releases/download/v0.9.3/react-relay-0.9.3.tgz",
+    "react-router": "^3.0.2",
```

Comment:
I’m not sold on client-side routing yet, I was fine with adding this for the purpose of being able to work on full pages, but I want us to focus on optimised server-side rendering and only later on sprinkle client-side optimisations where really needed. In my opinion it’s too early to know where that is, but adding those optimisations now immediately adds a lot of complexity and leads to even more (e.g. adding redux).